src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve('Promise 1 resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise 2 rejected!'));
```

Comment:
Same as above. There's no reason to use that here, if you put that to "message error-message" later.